### ISSUE TITLE
Files are not erased after exit. 

### ISSUE THREAD

**Issue by jgalati** [mafintosh/peerflix#256]
I'm running:
OS: Windows 8.1
Language: Spanish 
npm: v2.14.12
node: v4.2.4
peerflix 0.32.3

I stream with the -r option, but when I press control-c to stop streaming I get the following error and the files are not erased from the temp folder.

```
"dn" no se reconoce como un comando interno o externo,
programa o archivo por lotes ejecutable.
"tr" no se reconoce como un comando interno o externo,
programa o archivo por lotes ejecutable.
"tr" no se reconoce como un comando interno o externo,
programa o archivo por lotes ejecutable.
```

**Comment by joshterrill** [comment]
Similar issue was just opened a week or so ago. #252

**Comment by jgalati** [comment]
Yes, but the OS is different and I included the error log. I think that is not that the erase of the file fails rather than the program exits before it reaches the part where it should erase the file because of the error stated above.
